src/android/StatusBar.java

```diff
@@ -64,6 +64,9 @@ public void run() {
 
                 // Read 'StatusBarStyle' from config.xml, default is 'lightcontent'.
                 setStatusBarStyle(preferences.getString("StatusBarStyle", "lightcontent"));
+
+                // Read 'StatusBarOverlaysWebView' from config.xml, default is false.
+                setStatusBarTransparent(preferences.getBoolean("StatusBarOverlaysWebView", false));
```

Comment:
`setStatusBarTransparent` should be called before setting the background colour, or else the background colour will be ignored and the statusbar will always be fully transparent.

Reply:
I've raised an issue describing this at https://github.com/apache/cordova-plugin-statusbar/issues/159 because I think the statusbar plugin should be smarter in handling this. But for now, I agree, the workaround is to invoke `setStatusBarTransparent` before `setStatusBarBackgroundColor`.